eclipse_formatter.xml

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I use IntelliJ, mainly, so this file won't help very much.
There are some format guidelines which may be important (indentation, tabs, etc), but I think that the diff/merge tools are smart enough to manage this. If not, those guidelines should be defined in the README.md.